themes/website/templates/clients.html

```diff
@@ -319,7 +319,7 @@ <h1>OS support:</h1>
 						<th>Toxic</th>
 						<th class="danger">No</th>
 						<th class="success">Yes</th>
-						<th class="success">Yes</th>
+						<th class="success">Yes <sup id="ref-2"><a href="#note-2">[2]</a></sup></th>
```

Comment:
`id` has to be unique within the html document. I'd probably just drop the IDs and the links from under the table up to here.

Reply:
I've missed that. Done.